twist_mux:
  ros__parameters:
    output_topic: "cmd_vel"
    emergency_topic: "emergency_stop"
    emergency_timeout: 0.5
    enable_emergency: True
    inputs: ["teleop", "nav"]
    
    teleop:
      topic: "/cmd_vel_teleop"
      priority: 100
      timeout: 0.5
    
    nav:
      topic: "/cmd_vel_nav"
      priority: 50
      timeout: 0.5